build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -128,6 +128,20 @@ spec:
                   - type
                   type: object
                 type: array
+              lbSubnets:
```

Comment:
do you think we should add `vpcStatus` in front of lbSubnets

Reply:
added, thanks.